### ISSUE TITLE
ImportError: cannot import name 'get_gpg'

### ISSUE THREAD

**Issue by Euphorbium** [blag/django-secure-mail#1]
```
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 363, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 337, in execute
    django.setup()
  File "/usr/local/lib/python3.6/site-packages/django/__init__.py", line 27, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/local/lib/python3.6/site-packages/django/apps/registry.py", line 85, in populate
    app_config = AppConfig.create(entry)
  File "/usr/local/lib/python3.6/site-packages/django/apps/config.py", line 120, in create
    mod = import_module(mod_path)
  File "/usr/local/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/usr/local/lib/python3.6/site-packages/secure_mail/apps.py", line 4, in <module>
    from .checks import check_signing_key, SecureMailTags
  File "/usr/local/lib/python3.6/site-packages/secure_mail/checks.py", line 6, in <module>
    from secure_mail.utils import get_gpg
ImportError: cannot import name 'get_gpg'
```

python-gnupg and gnupg2 are installed.  Ubuntu.

**Comment by Euphorbium** [comment]
setting the SECURE_MAIL_GNUPG_HOME resolves this issue. Might be helpful to mention in the docs.

**Comment by blag** [comment]
Yes, specifying `SECURE_MAIL_GNUPG_HOME` is pretty much required for all cases, but that fact should be more clear in the documentation. I'll get to that soon. Thanks!

**Comment by blag** [comment]
Fixed in a3a248dc3e78f6d8cf23360588ec349e2df2098d.
